pkg/printers/internalversion/printers.go

```diff
@@ -1151,6 +1152,25 @@ func getNodeExternalIP(node *api.Node) string {
 	return "<none>"
 }
 
+// Returns the internal IP of the node or "<none>" if none is found.
+// If multiple internal IPs are found, ",+more" are appended after the first one.
```

Comment:
why not mirror the external IP code?

Reply:
Just as the [above discussion](https://github.com/kubernetes/kubernetes/pull/57623#discussion_r162105304) said, that will hide the information from a user that there are other IPs. In case of multiple, we should inform user there are more.

Reply:
a single value of "1.2.3.4,more" isn't usable as an IP... given the external IP only prints a single value, and kubelets only report a single internal IP, I'd prefer we just print the first

Reply:
Why, that will give you false information about IPs a node have.

Reply:
> a single value of "1.2.3.4,more" isn't usable as an IP

Right. This may result in parsing error for downstream projects.

> given the external IP only prints a single value, and kubelets only report a single internal IP, I'd prefer we just print the first

LGTM. Agree.